File: all_sort/bubble_sort.py
#!/usr/bin/python3

def bubble_sort(seq):
    n = len(seq)
    for i in range(n-1):
        for j in range(n-1-i):
            if seq[j] > seq[j+1]:
                seq[j], seq[j+1] = seq[j+1], seq[j]

import random
lst = list(range(10))
random.shuffle(lst)
print(lst)
bubble_sort(lst)
print(lst)

File: all_sort/go/bubble_sort.go
package main

import "fmt"

func bubble_sort(seq []int){
	l := len(seq)
	for i := 0; i < l-1; i++{
		for j := 0; j < l-i-1; j++{
		if seq[j] > seq[j+1]{
			seq[j], seq[j+1] = seq[j+1], seq[j]
		}
		}
	}
}

func main(){
	var lst = []int{1, 3, 2, 4, 5, 7, 6}
	bubble_sort(lst)
	fmt.Printf("%v", lst)
}
